File: app/src/prod/java/com/github/premnirmal/ticker/CrashLogger.java
package com.github.premnirmal.ticker;


import com.crashlytics.android.Crashlytics;

/**
 * Created by premnirmal on 7/5/15.
 */
public class CrashLogger {

    public static void logException(Throwable throwable) {
        Crashlytics.logException(throwable);
    }
}

File: app/src/prod/kotlin/com/github/premnirmal/ticker/CrashLogger.kt
package com.github.premnirmal.ticker

import com.crashlytics.android.Crashlytics

/**
 * Created by premnirmal on 2/26/16.
 */
class CrashLogger {

  companion object {
    fun logException(throwable: Throwable) {
      Crashlytics.logException(throwable)
    }
  }
}